### ISSUE TITLE
Cross Entropy Loss and Binary Cross Entropy Loss

### ISSUE THREAD

**Issue by 1611Dhruv** [RSYashwanth/goNN#5]
Same thing :)

https://en.wikipedia.org/wiki/Cross-entropy
